src/main/java/frc/robot/RobotContainer.java

```diff
@@ -27,14 +35,93 @@
+            new StraightDrive(driveSubsystem, 120, 0.6),
+            new TurnUsingGyro(driveSubsystem, -45),
+            new StraightDrive(driveSubsystem, 84.9, 0.6),
+            new TurnUsingGyro(driveSubsystem, 45),
+			new StraightDrive(driveSubsystem, 42.4, 0.6),
+            new TurnUsingGyro(driveSubsystem, 135),
+            new StraightDrive(driveSubsystem, 42.4, 0.6),
+            new TurnUsingGyro(driveSubsystem, -135),
+            new StraightDrive(driveSubsystem, 84.9, 0.6),
+            new TurnUsingGyro(driveSubsystem, 180),
+			new StraightDrive(driveSubsystem, 120, 0.6),
+            new TurnUsingGyro(driveSubsystem, 135),
+            new StraightDrive(driveSubsystem, 84.9, 0.6)
+		);
+	private SequentialCommandGroup bouncePath = 
+		new SequentialCommandGroup (
+			new TurnUsingGyro(driveSubsystem, 63.4),
+			new StraightDrive(driveSubsystem, 67.1, 0.6),
+            new TurnUsingGyro(driveSubsystem, -63.4),
+            new StraightDrive(driveSubsystem, 134.2, 0.6),
+            new TurnUsingGyro(driveSubsystem, 76.0),
+            new StraightDrive(driveSubsystem, 123.7, 0.6),
+            new TurnUsingGyro(driveSubsystem, -76.0),
+			new StraightDrive(driveSubsystem, 123.7, 0.6),
+            new TurnUsingGyro(driveSubsystem, 0),
+            new StraightDrive(driveSubsystem, 30, 0.6),
+            new TurnUsingGyro(driveSubsystem, 76.0),
+            new StraightDrive(driveSubsystem, 123.7, 0.6),
+            new TurnUsingGyro(driveSubsystem, -63.4),
+			new StraightDrive(driveSubsystem, 67.1, 0.6),
+            new TurnUsingGyro(driveSubsystem, 0)
+		);
+	private SequentialCommandGroup barrelPath = 
 		new SequentialCommandGroup (
-			new StraightDrive(driveSubsystem, 120, 0.3),
+			new StraightDrive(driveSubsystem, 114, 0.6),
+			new TurnUsingGyro(driveSubsystem, -90),
+			new StraightDrive(driveSubsystem, 60, 0.6),
+			new TurnUsingGyro(driveSubsystem, -90),
+			new StraightDrive(driveSubsystem, 60, 0.6),
+			new TurnUsingGyro(driveSubsystem, -90),
+			new StraightDrive(driveSubsystem, 60, 0.6),
+			new StraightDrive(driveSubsystem, 198, 0.6), 
+			new TurnUsingGyro(driveSubsystem, -90),
+			new StraightDrive(driveSubsystem, 60, 0.6),
+			new TurnUsingGyro(driveSubsystem, -90),
+			new StraightDrive(driveSubsystem, 60, 0.6),
+			new TurnUsingGyro(driveSubsystem, -90),
+			new StraightDrive(driveSubsystem, 120, 0.6),
+			new TurnUsingGyro(driveSubsystem, 90),
+			new StraightDrive(driveSubsystem, 60, 0.6),
 			new TurnUsingGyro(driveSubsystem, 90),
-			new StraightDrive(driveSubsystem, 60, 0.3)
-			// new TurnUsingGyro(driveSubsystem, 45),
-			// new StraightDrive(driveSubsystem, 36, 0.3)
+			new StraightDrive(driveSubsystem, 60, 0.6), 
+			new TurnUsingGyro(driveSubsystem, -90), 
+			new StraightDrive(driveSubsystem, 252, 0.6)
 		);
+	 private final Command autonomousCommand = 	
+	 	new SequentialCommandGroup(
+	 		new ParallelCommandGroup (
+				new EnableIntake(intakeStorageSubsystem),
```

Comment:
Shouldn't enableIntake happen in sequence before the other drive commands? We want the intake to be deployed and up to speed before we start driving.